lib/routes/changes.js

```diff
@@ -75,7 +75,14 @@ module.exports = function (app) {
                 console.log(err);
                 res.end();
                 cleanup();
+              }).on('complete', function () {
+                cleanup();
               });
+
+            req.connection.on('close', function () {
+              console.log('CLOSE!')
+              changes.cancel()
+            });
```

Comment:
shall we add the same thing after line 53 for when `req.query.feed === 'continuous'`? I’m not sure if `req.connection.on('close', cb)` is the right thing to do here, but it fixed my problem